src/constants.ts

```diff
@@ -30,6 +30,8 @@ export const headerNames = {
   arnsResolvedId: 'X-ArNS-Resolved-Id',
   arnsProcessId: 'X-ArNS-Process-Id',
   arnsResolvedAt: 'X-ArNS-Resolved-At',
+  arnsLimit: 'X-ArNS-Limit',
+  arnsIndex: 'X-ArNS-Index',
```

Comment:
nit - could change them to 
```suggestion
  arnsLimit: 'X-ArNS-Undername-Limit',
  arnsIndex: 'X-ArNS-Record-Index',
```

Reply:
@dtfiedler Agreed. I think it's a good idea to change these.

Reply:
Agreed!

Reply:
[ba0ef63](https://github.com/ar-io/ar-io-node/pull/234/commits/ba0ef6301ed876517dd14a30791918be121bda09)